app/models/journeys/additional_payments_for_teaching/session.rb

```diff
@@ -0,0 +1,11 @@
+module Journeys
+  module AdditionalPaymentsForTeaching
+    class Session < Journeys::Session
+      attribute :answers, SessionAnswersType.new
+
+      def journey_module
+        Journeys::AdditionalPaymentsForTeaching
+      end
```

Comment:
I'm not sure if this is necessary as this class lives in the relevant journey namespace. If we still need the class to tell us this (I'm not sure where it's used) we could potentially implement it in `ClaimSubmissionBaseForm` using `Module.nesting.first`.